frontend/src/components/Footer/Footer.css

```diff
@@ -0,0 +1,47 @@
+.designBySection {
+    font-size: 1vw;
+    opacity: 0.5;
+    color: #757373;
+    text-align: center;
+    padding-top: 12px;
+    font-weight: 500;
+}
+
+.designByCo {
+    font-size: 1vw;
+    color: #1f1e1e;
+    font-weight: 600;
+   /* padding-bottom: 5vw;*/
```

Comment:
I think commented out code is unnecessary clutter, let's get rid of it. We can always add it back in if we decide we need it.

Reply:
Yes. Thanks. Forgot to remove.